### PR TITLE
feat: Update Firefox support to 78

### PR DESCRIPTION
**Pull request by castastrophe**
## Infrastructure updates

RHEL CSB has updated to Firefox v78 so we can bump our support!

### Testing instructions

Be sure to include detailed instructions on how your update can be tested by another developer.

1. Pull up Firefox 78 and do a light regression test.

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable to your PR.

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Repository compiles and tests pass.
- [x] Changelog updated.
- [x] Documentation updated or added, if necessary.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

**Comment by castastrophe** [comment]
I pulled this up on BrowserStack on FF 78 and I didn't see any regressions

**Comment by heyMP** [comment]
I created a new element using this branch.  I ran into some storybook errors, I don't think they are related though:

Screen Shot 2021-01-25 at 5 30 56 PM

The new package.json did contain the engines and browserlist settings:

Screen Shot 2021-01-25 at 5 32 56 PM

**Comment by heyMP** [comment]
Firefox Regression:

Looks like pfe-accordian is broken 😬

Screen Shot 2021-01-25 at 5 43 38 PM

**Comment by castastrophe** [comment]
@heyMP I updated a few packages that support our autoprefixing and I'm seeing much better rendering results.  Should be good to go now.  Let me know if you still see these issues!

**Review by heyMP — APPROVED** [review]
Regression tested.

File: scripts/gulpfile.factory.js [hunk 1]
```diff
@@ -5,9 +5,7 @@ module.exports = function factory({
 } = {}) {
   elementName = elementName.replace(/s$/, "");
   const { task, src, dest, watch, parallel, series } = require("gulp");
-  const sassdoc = require("sassdoc");
-
-  const browser_support = ["last 2 versions", "Firefox >= 51", "iOS >= 8", "ie 11"];
+  // const sassdoc = require("sassdoc");
```
[the remark this answers is not in the document]
Comment (heyMP) [reply scripts/gulpfile.factory.js:8]: Per CR: "None of the components have sassdoc so it was throwing an unused error"

**Pull request merged**